plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -340,6 +343,8 @@ function FakeCover:init()
 end
 
 
+local progress_widget
```

Comment:
Declare it just after local corner_mark at line 45: it's the same kind of stuff, deserves the same kind of comment explaining it's unique and re-used.